docs/specification.md

```diff
@@ -146,11 +158,19 @@ Some configuration values are specified in runtime and can be changed during the
 They must be provided on origination to construct the contract's initial storage.
 These values are:
 1. `admin : address` is the address that can perform administrative actions.
-2. `governance_token` is the FA2 contract address/token id pair that will be used as the governance token.
-3. `voting_period : nat` specifies how long the voting period lasts in seconds.
-4. `quorum_threshold : quorum_threshold` specifies what fraction of the frozen
+2. `guardian : address` is the address of a contract that have permission to call `drop_proposal`.
+    - `guardian` contract cannot initiate the transaction that results in a call to `drop_proposal`
+3. `governance_token` is the FA2 contract address/token id pair that will be used as the governance token.
+4. `voting_period : nat` specifies how long the voting period lasts in seconds.
+5. `proposal_flush_time : nat`
+    - Specifies how long in seconds after the proposal is proposed such that it can be flushed.
+    - IMPORTANT: Must be bigger than `voting_period * 2`.
+6. `proposal_expired_time : nat`
+    - Specifies how long in seconds after the proposal is proposed such that it can be considered as expired.
```

Comment:
```suggestion
    - Specifies in seconds how long after the proposal is proposed it is considered expired.
```
This is less of a nitpick as after this time it isn't just that it **can** be considered expired, but it **has** to.